core/src/SYCL/Kokkos_SYCL_Instance.cpp

```diff
@@ -75,7 +75,15 @@ void SYCLInternal::initialize(const sycl::device& d) {
       Kokkos::Impl::throw_runtime_exception(
           "There was an asynchronous SYCL error!\n");
   };
-  initialize(sycl::queue{d, exception_handler});
+  // FIXME SYCL+Cuda Apparently, submit_barrier doesn't quite work as expected
+  // for oneAPI 2023.0.0 on NVIDIA GPUs.
```

Comment:
Was this reported?  How do we keep track on progress on this?

Reply:
I don't really have a good reproducer (yet). I just found in #6064 that replacing a `submit_barrier` with a `depends_on` fixed issues I was seeing only for `SYCL+Cuda`. I can try to come up with something to report to `intel/llvm`.